### ISSUE TITLE
Find a way around for enabling all the youtube videos to work in H2

### ISSUE THREAD

**Issue by Anonymous26** [poush/H2#69]
Few youtube videos aren't available in H2 right now due to some license reason. We need to find a way for enabling all the videos in H2 if possible. This issue is for discussion purposes, feel free to suggest idea to achieve the goal.

![h2_issue](https://user-images.githubusercontent.com/20697219/47221650-9c14c580-d3d2-11e8-96c4-4e27a19377bc.jpg)

**Comment by rouzbehhz** [comment]
@Anonymous26 @poush 
Just did some investigating and it won't allow embedding if the embedded player is trying to play a video that has "Music in video"
<img width="657" alt="screen shot 2018-10-19 at 4 55 29 pm" src="https://user-images.githubusercontent.com/40371763/47243379-cdd06100-d3bf-11e8-9b04-1969cb8e5979.png">

I tried embedding it in CodePen and it shows up fine: https://codepen.io/anon/pen/WayvvO

So if embedded on a web server, it'll allow streaming but not on an electron app. Tried using a chrome user agent with no avail. Will take a look later and see if there is a workaround or if they've disabled this feature as they're trying to push Youtube Music and a lot of spotify clones rely on embedding youtube music videos hidden which goes against YouTube's terms but people still do it. (We might need to skim through the terms for youtube).
